src/phase1.hpp

```diff
@@ -598,7 +598,8 @@ std::vector<uint64_t> RunPhase1(
     uint32_t const stripe_size,
     uint8_t const num_threads,
     bool const enable_bitfield,
-    bool const show_progress)
+    bool const show_progress,
+    bool verbose)
```

Comment:
I think 3 "naked" bools as function arguments exceed the limit of good readability.

I think it would make sense to replace these `bools` with a single `flags` argument instead. That way the call sites will tell you which features are enabled.

Reply:
I'm ok with that, I can make another PR for that, or I can try to do the changes here (But I prefer to only change 1 thing at a time per PR).

Reply:
In that case I would prefer that PR land first

Reply:
Sure, no problem, I'll do that or first.